fesvr/htif.cc

```diff
@@ -34,6 +34,12 @@
 # define TARGET_DIR "/" TARGET_ARCH "/bin/"
 #endif
 
+#ifdef __GNUC__
+# define UNUSED __attribute__ ((unused))
+#else
+# define UNUSED
+#endif
+
```

Comment:
Given this is source code and not a header, is there actually anything wrong with using the macro definitions in `../riscv/common.h` instead?

Reply:
Good call, that makes this simpler